scanctl/whitesource.py

```diff
@@ -93,8 +93,8 @@ def delete(self):
 @attr.s
 class FsAgent:
     token = attr.ib()
-    jar = attr.ib(os.path.join(os.getcwd(), 'bin', 'fs-agent.jar'))
-    config = attr.ib(os.path.join(os.getcwd(), 'etc', 'whitesource.config'))
+    jar = attr.ib(os.path.join(os.getcwd(), 'fs-agent.jar'))
+    config = attr.ib(os.path.join(os.getcwd(), 'whitesource.config'))
```

Comment:
It'd be helpful to me to have the keyword names for the arguments passed into `attr.ib`; it's not easy to remember what order arguments that `attr.ib` takes, made more difficult by the fact that it's an external lib. For For instance, is `os.path.join(os.getcwd(), 'fs-agent.jar')` a default? or an init value?

Reply:
This feels a bit overzealous. Are you also unsure what the argument name to `os.path.join` is? One should be able to assume basic familiarity to the libraries in use. I agree with you when in principle, especially if passing >= 3 args, say. But for passing one argument, in a function this common, I don't see the point.